.gitignore

```diff
@@ -1,3 +1,7 @@
+# Created by https://www.toptal.com/developers/gitignore/api/python,visualstudiocode
```

Comment:
Why do you keep changing all of the `.gitignore` files?

Reply:
I believe it was automated, I haven't touched the gitignore in a while